codegen/kuberesource/crd.go

```diff
@@ -27,7 +46,7 @@ func CustomResourceDefinitions(group model.Group) (objects []metav1.Object, err
 			}
 		}
 
-		objects = append(objects, CustomResourceDefinition(resource, validationSchema))
+		objects = append(objects, CustomResourceDefinition(resource, validationSchema, opts...))
```

Comment:
instead of creating/passing the extra `opts...` here, could we not just read `group.SkipSpecHash`?